### PR TITLE
Added first pass at docs

### PR DESCRIPTION
**Pull request by brettwold**

### CODE REVIEW

File: documentation/LICENSE [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Copyright 2016 Marcel Pociot
```
Comment (louis-barman) [inline documentation/LICENSE]: Who is this?

File: documentation/LICENSE [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+Copyright 2008-2013 Concur Technologies, Inc.
```
Comment (louis-barman) [inline documentation/LICENSE]: +Copyright 2017 AEVI??

File: documentation/LICENSE (hunk 1 above)
Comment (brettwold) [reply documentation/LICENSE]: This is the license for whiteboard our docs generator

File: documentation/LICENSE (hunk 2 above)
Comment (brettwold) [reply documentation/LICENSE]: As above

**Pull request merged**